2022-polls/MapLibre-Governing-Board-Election-Ballots.txt

```diff
@@ -0,0 +1,45 @@
+7 5
```

Comment:
What are these numbers?

Reply:
I wondered too. I don't know, was just the opavote output...